aclk/aclk_stats.c

```diff
@@ -364,7 +364,7 @@ void aclk_stats_upd_online(int online) {
 }
 
 #ifdef NETDATA_INTERNAL_CHECKS
-static usec_t pub_time[UINT16_MAX];
+static usec_t pub_time[UINT16_MAX] = {0};
```

Comment:
this was copied from old ACLK but looking at it now it should really be `[UINT16_MAX+1]` as